src/vellum/workflows/nodes/displayable/api_node/tests/test_api_node.py

```diff
@@ -32,3 +36,32 @@ class SimpleBaseAPINode(APINode):
     bearer_token = vellum_client.execute_api.call_args.kwargs["bearer_token"]
     assert bearer_token == ClientVellumSecret(name="secret")
     assert terminal.headers == {"X-Response-Header": "bar"}
+
+
+def test_api_node_raises_error_when_api_call_fails(vellum_client):
+    # Mock the vellum_client to raise an ApiError
```

Comment:
[nit] We like to keep our tests to conforming to the `GIVEN`/`WHEN`/`THEN` comment pattern